packages/core/react/use-assistant.ts

```diff
@@ -39,13 +39,21 @@ export type UseAssistantHelpers = {
   error: undefined | unknown;
 };
 
+export type UseAssistantOptions = {
+  api: string;
+  threadId?: string | undefined;
+  credentials?: RequestCredentials;
+  headers?: Record<string, string> | Headers;
+  body?: object;
+};
+
 export function experimental_useAssistant({
   api,
   threadId: threadIdParam,
-}: {
-  api: string;
-  threadId?: string | undefined;
-}): UseAssistantHelpers {
+  credentials,
+  headers = {},
+  body = {},
```

Comment:
`useChat` does not init those parameters. I'd prefer to match the style from `useChat` as much as possible.

Reply:
Approved assuming this change comes in.

Reply:
Updated